lexicons/com/atproto/sync/defs.json

```diff
@@ -0,0 +1,22 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.defs",
+  "defs": {
+    "takendown": {
+      "type": "token",
+      "description": "Repo hosting status indicating that an administrator has taken down the repo, for a permanent period (though this may be reversed)."
+    },
+    "suspended": {
+      "type": "token",
+      "description": "Repo hosting status indicating that an administrator has taken down the repo, for a limited (but possibly indefinite) time period."
+    },
+    "deleted": {
+      "type": "token",
+      "description": "Repo hosting status indicating that the repository has been removed. The repo may be re-opened or migrated back to this host in the future, but the contents have been deleted for now. Does not clarify if the account self-deleted or an administrator or operator intervened."
+    },
+    "deactivated": {
+      "type": "token",
+      "description": "Repo hosting status indicating that the repository has been pause and should not be re-distributed, usually on request of the account holder. This may be temporary or indefinite."
```

Comment:
s/pause/paused/ (though I'm not sure if "repository has been paused" is the best word here...)